tests/show_techsupport/test_techsupport.py

```diff
@@ -264,7 +264,11 @@ def execute_command(duthost, since):
     :param duthost: DUT
     :param since: since string enterd by user
     """
-    result = duthost.command("show techsupport -r --since={}".format('"' + since + '"'), module_ignore_errors=True)
+    opt = "-r" if duthost.sonic_release not in ["201811", "201911"] else ""
```

Comment:
Hi Mahesh, I personally find it a little hard to understand this line.
Might be better if:
opt = "" if duthost.sonic_release in ["201811", "201911"] else "-r" ?

Reply:
agree, next time will keep in mind